docs/learning/README.md

```diff
@@ -1,41 +1,86 @@
        These decks from October 2021 might also help:
-       - [Utility](https://docs.google.com/presentation/d/1NU0PnegtBm5ioLu0VQMiluWT4usHnavDKrGvS3p8QdM/edit)
-       - [Persistence](https://docs.google.com/presentation/d/1qDA-pRMT1KV9byUAU49bvd_5seaILPAh6i3vA7j5l8o/edit)
-       - [P2p](https://docs.google.com/presentation/d/1CLeAcGJbM_iP76vnCoHreU1chB9vFWIYWAwQHa-MPbc/edit)
-       - [Consensus](https://docs.google.com/presentation/d/18CtSxxLLHY1N7HEJtja633mVF1_a9blaE2fe2-WgGAo/edit)
-    -->
+         - [Utility](https://docs.google.com/presentation/d/1NU0PnegtBm5ioLu0VQMiluWT4usHnavDKrGvS3p8QdM/edit)
+         - [Persistence](https://docs.google.com/presentation/d/1qDA-pRMT1KV9byUAU49bvd_5seaILPAh6i3vA7j5l8o/edit)
+         - [P2p](https://docs.google.com/presentation/d/1CLeAcGJbM_iP76vnCoHreU1chB9vFWIYWAwQHa-MPbc/edit)
+         - [Consensus](https://docs.google.com/presentation/d/18CtSxxLLHY1N7HEJtja633mVF1_a9blaE2fe2-WgGAo/edit)
+   -->
 
-6.  To start getting acquainted with the code, look at the [README on the shared architecture](https://github.com/pokt-network/pocket/tree/main/shared).
+6. Start getting acquainted with the code structure by looking at the [docs on the shared architecture](https://github.com/pokt-network/pocket/tree/main/shared). _(1 hour)_
 
-7.  Want to get a sense of all the open issues and tickets? [Check out this page](https://github.com/orgs/pokt-network/projects/142/views/12) or ask for a starter task in the [#v1-dev](https://discord.com/channels/553741558869131266/986789914379186226) discord channel if you don’t have one.
+   - This will help you understand the code architecture of Pocket V1
 
-8.  Jump on a call and pair code!
+7. View our [V1 Roadmap](https://github.com/pokt-network/pocket/blob/main/docs/roadmap/README.md). _(10 mins)_
+
+   - This will give you insight into our development & release timelines
+
+8. Get a sense of all the open issues and tickets [in out Github project](https://github.com/orgs/pokt-network/projects/142/views/12). _(1 hour)_
+9. If you don't already have a starter task, pick one from [Dework](https://app.dework.xyz/pokt-network), our [open issues](https://github.com/pokt-network/pocket/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc) or ask the team in the [#v1-dev](https://discord.com/channels/553741558869131266/986789914379186226) discord channel. _(1 hour)_
+
+10. Jump on a call and pair code! _(∞ and beyond)_
 
 - If you need a walk-through of the code and some pointers before getting started, **jump on a call!**
 - If you need to take your time to understand the problem and the code first, do so, and then **jump on a call!**
 - If you don't need help, **when you’re about 33% of the way done**, show your draft work and get some feedback, so **jump on a call!**
 - Just leave a message in the [#v1-dev](https://discord.com/channels/553741558869131266/986789914379186226) discord channel and someone from the core team will respond.
 
+## Technical Foundation
```

Comment:
I recommend to add some emoji before/after each H1 (section name) , cause they are a little bit hard to read/distinguish from other smaller texts.